### ISSUE TITLE
IE11 breaks when a static method named `length` is used.

### ISSUE THREAD

**Issue by anilanar** [babel/babel#9950]
## Bug Report

**Current Behaviour**
IE11 breaks when a static method named `length` is used.

For example, the following class:

```
class Foo {
  static length() { return 10; }
}
```

fails to work in IE11 (using `preset-env`) because apparently `length` is a non-configurable property of function.

**Input Code**
https://babeljs.io/repl/#?babili=false&browsers=ie%2011&build=&builtIns=false&spec=false&loose=false&code_lz=MYGwhgzhAEBiD29oG8BQ1oQC5iwS2GhAFMA7AcywAsAKAShXWgF9Vmg&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=false&fileSize=false&timeTravel=false&sourceType=script&lineWrap=false&presets=env&prettier=false&targets=&version=7.4.4&externalPlugins=

**Expected behaviour/code**
I expect Babel to throw an error during transformation if someone uses a static property/method named `length`. This can be further extended to cases in which babel uses `defineProperty` and the target object has known non-configurable properties.

**Screenshots**

![image](https://user-images.githubusercontent.com/753736/57227989-fd208580-7012-11e9-9607-2fed151ef02b.png)

![image](https://user-images.githubusercontent.com/753736/57228278-b2533d80-7013-11e9-95f1-1c18e1ff1e99.png)

**Comment by nicolo-ribaudo** [comment]
```js
class Foo {
  static length() { return 10; }
}
```
is valid JavaScript code, so Babel should accept it. I think that it's a linter's duty to disallow it.

**Comment by anilanar** [comment]
I disagree. Babel is supposed to take valid Javascript and output valid Javascript. If it cannot do so, due to technical limitations, it should fail. The output of static method `length` is invalid in ES5 because `length` is non-configurable.

**Comment by anilanar** [comment]
I'll close this as IE11 is dead.
